src/Persistence/Mapping/Exception/MappingException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Doctrine\Persistence\Mapping\Exception;
+
+use Doctrine\Persistence\Mapping\MappingException as LegacyMappingException;
+
+/**
+ * Will become an interface in 3.0.
```

Comment:
if you make a new name, why not making it an interface directly ? Then, each exception class can extend the legacy class and implement the new interface.

Reply:
Ah yeah, that would be better indeed, I think, I should probably have done it that way for the ORM.

Reply:
I agree that we should make this class an interface already. This would allow ORM 2.x to implement it already.